root/etc/s6-overlay/s6-rc.d/init-hishtory-config/run

```diff
@@ -4,15 +4,16 @@
 if [[ -n ${HISHTORY_POSTGRES_DB} ]]; then
     DBHOST=$(echo "${HISHTORY_POSTGRES_DB}" | awk -F '@' '{print $NF}' | awk -F ':' '{print $1}')
     DBPORT=$(echo "${HISHTORY_POSTGRES_DB}" | awk -F '@' '{print $NF}' | awk -F ':' '{print $2}' | awk -F '/' '{print $1}')
+    echo "Waiting for DB ${DBHOST} to become available on port ${DBPORT}..."
+    while :; do
+        if nc -w 1 -z "$DBHOST" "$DBPORT" > /dev/null 2>&1; then
+            exit 0
+        fi
+        sleep 3
+    done
+elif [[ -n ${HISHTORY_SQLITE_DB} ]]; then
+    echo "Using SQLite database, ensure you have mounted ${HISHTORY_SQLITE_DB} into the container"
```

Comment:
`ensure you have mounted ${HISHTORY_SQLITE_DB}` implies that you can mount this and not have docker create the folder. would it be safer to basename the variable for the message?

Reply:
Good point